deploy/jiva-csi.yaml

```diff
@@ -281,11 +281,19 @@ spec:
             # This count has been set to 10 for sanity test cases as it takes
             # time in minikube
             - "--retrycount=10"
+            # metricsBindAddress is the TCP address that the controller should bind to
+            # for serving prometheus metrics. Change the port if 9505 is already in use.
+            # Remove the flag to disable prometheus metrics.
+            - "--metricsBindAddress=$(OPENEBS_POD_IP):9505"
           env:
             - name: OPENEBS_NODE_ID
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: OPENEBS_POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
             - name: OPENEBS_CSI_ENDPOINT
```

Comment:
This is not required since pod will share the host network. So any request on the node will be routed to localhost.